app/policies/rule_policy.rb

```diff
@@ -3,11 +3,13 @@
 # Policies for accessing Rules
 class RulePolicy < ApplicationPolicy
   def index?
-    record.profiles.pluck(:account_id).include? user.account_id
+    record.profiles.pluck(:account_id).include?(user.account_id) ||
+      record.canonical?
```

Comment:
@dLobatog idk what I was thinking before :laughing: This should be correct and also easier to understand.